docs/docsite/rst/community/communication.rst

```diff
@@ -94,6 +94,14 @@ IRC meetings
 The Ansible community holds regular IRC meetings on various topics, and anyone who is interested is invited to
 participate. For more information about Ansible meetings, consult the `meeting schedule and agenda page <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
 
+The Bullhorn
+------------
+The Bullhorn is our newsletter for the Ansible developer community. If you have any questions or content you'd like to share, please reach out to us at the-bullhorn@redhat.com, or directly contribute/suggest content for upcoming issues. The Ansible News Working Group sources, curates, and promotes existing Ansible-related content, and encourages the creation of Ansible-related content across a variety of mediums. Initial curation and promotion goals are an Ansible Weekly Newsletter; future work may include Ansible Magazine (online site, Ansible-community created / edited); Ansible Developer Newsletter (less frequent, info that is relevant to contributors).
```

Comment:
@DarrenHSKG a bit of advice: from the maintainability/readability standpoint, it's better to have multiple shorter lines in the source RST rather than one very long line. It'll be rendered the same but would be a bit more pleasant to read in the source.

Reply:
Agree with @webknjaz, plus I don't think the part about Ansible News WG is needed here. This is just about The Bullhorn, so the first two sentences will suffice.

Link for "contribute/suggest content" https://github.com/ansible/community/issues/546

Reply:
This comment is true so long as the line is a full sentence. For Ansible anyway, the rst files are being translated and ending a line in the middle of a sentence causes translation problems. (Imagine ending the first part of this with a new line after `If you have any`.  The translation engine needs to see the full sentence and can't currently handle it when the sentence has a newline in the middle of it.